src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -3652,14 +3886,14 @@ protected boolean recompileJava(Collection<File> javaFilesChanged, List<String>
             }
             if (compileResult) {
                 if (tests) {
-                    info("Tests compilation was successful.");
+                    info(formatMessage("Tests compilation was successful", projectName));
```

Comment:
Generally we should avoid using methods to format messages, so that it will be easier to switch to localized messages later on.  Use separate message strings for the different situations.  Same for the other messages in this method.